src/dotnet-aspnet-codegenerator/project.json

```diff
@@ -7,15 +7,20 @@
     "Microsoft.Extensions.CodeGeneration": "1.0.0-*",
     "Microsoft.Extensions.CommandLineUtils": "1.0.0-*",
     "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
-    "NETStandard.Library": "1.5.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
   "frameworks": {
-    "netstandardapp1.5": {
```

Comment:
I thought to make it a portable app, the tfm was supposed to be netstandard1.5 as per 
https://github.com/dotnet/cli/blob/rel/1.0.0/TestAssets/TestProjects/PortableTests/PortableApp/project.json

Isn't this eventually going to be netcoreapp? 
Not sure if it already needs to be netcoreapp?